docs/UserGuide.md

```diff
@@ -164,96 +136,101 @@ Item | Description
+* `( | )` : Items in round brackets `()` separated by `|` means that you can only choose 1 of the items <br>
+  * e.g. `([t/TAG] | [ta/TAG_TO_ADD] [t/TAG_TO_DELETE])` can be used as `t/condo` or `ta/condo td/small condo` but not `t/condo ta/condo` or `t/condo td/small condo`
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   * e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 ------------------
 
 ## Valid properties/buyers
+This section describes the requirements of certain fields.
 
-Here are some fields that are shared amongst commands.
-
-### Property/Buyer names
-* Names must start with a letter or number, and only contain alphanumerical characters, spaces and hyphens (`-`).
+### Property/buyer names
+* Names must only contain alphanumerical characters, spaces and hyphens (`-`).
 * Names have a maximum allowed length of 50.
 * Buyers with identical names are not allowed.
 
 ### Property addresses
-* Addresses must start with a letter or number, and only contain alphanumerical characters, and the following special characters:
-  * spaces,
-  * hyphens (`-`),
-  * commas (`,`),
-  * semicolons (`;`),
-  * and hashes (`#`).
+* Addresses must only contain alphanumerical characters or the following special characters:
+  * spaces
+  * hyphens (`-`)
+  * commas (`,`)
+  * semicolons (`;`)
+  * hashes (`#`)
 * Addresses have a maximum allowed length of 100.
 * Properties with identical addresses are not allowed.
 
-### Property Prices/Buyer Budget
-
-* Prices must be between 4 and 9 digits (both inclusive).
+### Property prices and buyer budget
+* Prices must be numbers between 4 and 9 digits (both inclusive).
 * Leading zeroes will be ignored. For e.g., `00100` has 5 characters, but it only has 3 digits, not counting the leading 0s. Hence, `00100` is an invalid price.
 
 ### Phone numbers
-* Phone number should only contain alphanumeric characters, hyphens (`-`), parentheses (`()`), plus signs (`+`) and spaces.
-* Phone numbers must have at least 3 characters, excluding leading and trailing spaces
+* Phone numbers should only contain alphanumeric characters, hyphens (`-`), parentheses (`()`), plus signs (`+`) and spaces.
+* Phone numbers must have at least 3 characters, excluding leading and trailing spaces.
   * Valid: 
     * `9123 4567`
     * `(office) +65 6123 4567`
     * `nil`
-  * Invalid: 
-    * <code>&nbsp;&nbsp;&nbsp;a&nbsp;&nbsp;&nbsp;</code> (Excluding leading and trailing spaces, the phone number only contains 1 character)
+  * Invalid:
     * `#123` (contains invalid hash symbol)
 
 ### Tags
 * Tags are always optional.
-* Tags must start with a letter or number, and only contain alphanumerical characters, spaces and hyphens (`-`).
+* Tags must start with a letter or number, and only contain alphanumeric characters, spaces and hyphens (`-`).
```

Comment:
Are we standardizing to alphanumeric or alphanumerical because in the names and address section, it is alphanumerical

Reply:
Alphanumeric is the more standard term, so we should switch to alphanumeric

Reply:
will standardise to alphanumeric